docker/backend-dev/conf/Config.pm

```diff
@@ -553,17 +690,61 @@ $options{import_export_fields_groups} = [
 
 %weblink_templates = (
 
-	'wikidata:en' => { href => 'https://www.wikidata.org/wiki/%s', text => 'Wikidata', parse => sub
-	{
-		my ($url) = @_;
-		if ($url =~ /^https?:\/\/www.wikidata.org\/wiki\/(Q\d+)$/) {
-			return $1
+	'wikidata:en' => {
+		href => 'https://www.wikidata.org/wiki/%s',
+		text => 'Wikidata',
+		parse => sub {
+			my ($url) = @_;
+			if ($url =~ /^https?:\/\/www.wikidata.org\/wiki\/(Q\d+)$/) {
+				return $1
+			}
+
+			return;
 		}
+	},
+);
 
-		return;
-	} },
+# allow moving products to other instances of Product Opener on the same server
+# e.g. OFF -> OBF
+
+$options{current_server} = "off";
+
+$options{other_servers} = {
+	obf =>
+	{
+		name => "Open Beauty Facts",
+		data_root => "/srv/obf",
```

Comment:
These don't exist in the docker environment, though.

Reply:
Well it doesn't really hurt. I'd like to keep the Config.pm file as close as possible to the Config_off.pm file. Ideally they should be the same. Maybe we can just move anything that needs to be different to Config2.pm (like the google analytics stuff)

Regarding the "admin" user, is there such a user?

Reply:
> Well it doesn't really hurt.

Doesn't it cause an unusable functionality for admin users?

> I'd like to keep the Config.pm file as close as possible to the Config_off.pm file. Ideally they should be the same. Maybe we can just move anything that needs to be different to Config2.pm (like the google analytics stuff)

How much of that is specific to _off, _obf_, etc.? 

> Regarding the "admin" user, is there such a user?

There are no users given by default, so you can register an admin with whatever password you want.

Reply:
> How much of that is specific to _off, _obf_, etc.?

I think the docker environment should by default use Config_off.pm, which turns all the specific OFF feature on.

Reply:
> Doesn't it cause an unusable functionality for admin users?

It's going to enable the feature that allows admin users to move a product to OBF etc. if they enter "obf" as the new barcode. And it will not work, but that's not going to cause an issue. On the contrary, it will force us to handle the case gracefully when the configuration specifies paths etc. that don't actually exist.